src/views/Categories.vue

```diff
@@ -5,7 +5,31 @@
     </div>
     <input type="text" v-model="newCategoryName">
     <button @click="addCategory">Add category</button>
+    <button data-bs-toggle="modal" data-bs-target="#staticBackdrop"
+    @click="deleteCategoryControl">Delete category</button>
```

Comment:
deleteCategoryControl is not a great name. A better name / function signature is deleteCategory(categoryName). Better to make button disabled if category cannot be deleted and show a tooltip (check bootstap docs, you'll need js parts of bootstrap too) which says something like `this category cannot be deleted since it contains questions` would be better